libraries/AP_EFI/AP_EFI_Backend.h

```diff
@@ -35,16 +35,16 @@ class AP_EFI_Backend {
     // Copies internal state to the frontend state
     void copy_to_frontend();
 
-    // Semaphore for access to shared frontend data
-    HAL_Semaphore sem;
-
     // Internal state for this driver (before copying to frontend)
     EFI_State internal_state;
 
     int8_t get_uavcan_node_id(void) const;
     float get_coef1(void) const;
     float get_coef2(void) const;
 
+    AP_EFI_Backend *get_driver(void);
```

Comment:
This is wrong?  New dangling declaration....

Reply:
removed